### PR TITLE
docs: add upgrade docs for 1.36

### PR DESCRIPTION
**Pull request by nikophil**
NB: I.ve not updated the docs: I'd like to try to create a psalm extension before. The docs will differ depending on if it is possible to do the same thing for psalm than for phpstan

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -213,7 +213,7 @@ This command will generate a ``PostFactory`` class that looks like this:
 
 .. note::
 
-    The generated ``@method`` docblocks above enable autocompletion with PhpStorm but
+    The generated ``@method`` docblocks above enable autocompletion with you IDE but
```
Comment (kbond) [inline docs/index.rst]:
```suggestion
    The generated ``@method`` docblocks above enable autocompletion with your IDE but
```

**Comment by kbond** [comment]
Update to include psalm plugin once #469 is merged?

**Comment by nikophil** [comment]
yep, that's was I was planning :)

**Comment by nikophil** [comment]
still needed here: upgrade doc with `ObjectFactory` and `PersistenceObjectFactory`

File: docs/index.rst [hunk 2]
```diff
@@ -1972,6 +1942,37 @@ Objects can be fetched from pools in your tests, fixtures or other stories:
...
+Or you can add the following snippet to `psalm.xml`:
+
+.. code-block:: xml
+
+    <plugins>
+        <pluginClass class="Zenstruck\Psalm\FoundryPlugin" />
+    </plugins>
+
+
+
```
Comment (OskarStark) [inline docs/index.rst:1975]:
```suggestion
```

**Comment by OskarStark** [comment]
As 1.36 is released, should this PR be merged?

**Comment by kbond** [comment]
Not yet, this PR and #462 is on hold for now.

**Pull request closed without merging**